web/src/ApiButton.tsx

```diff
@@ -72,20 +76,21 @@ export type ButtonSet = {
 }
 
 export enum ApiButtonType {
-  Nav = "Global",
+  Global = "Global",
   Resource = "Resource",
 }
 
-enum ApiButtonToggleState {
+export enum ApiButtonToggleState {
   On = "on",
   Off = "off",
 }
 
 // Constants
 export const UIBUTTON_SPEC_HASH = "uibuttonspec-hash"
 export const UIBUTTON_ANNOTATION_TYPE = "tilt.dev/uibutton-type"
+export const UIBUTTON_NAV_COMPONENT_ID = "nav"
```

Comment:
Nit: it looks like other places in this PR are changing `Nav` -> `Global`, should this be `UIBUTTON_GLOBAL_COMPONENT_ID`?